3-plot_subtype_kappa.R

```diff
@@ -77,3 +78,13 @@ for (cls in cls.methods) {
     theme(axis.text.x = element_text(angle = 45, vjust = 0.5)) 
   ggsave(plot.nm, plot = last_plot(), height = 3.5, width = 15)
 }
+
+# get summary data.frame + write to file
+summary.df <- 
+  kappa.master.df %>%
+    dplyr::group_by(Classifier, Normalization, Platform, Perc.seq) %>%
+    dplyr::summarise(Median = median(Kappa),
+                     Mean = mean(Kappa),
+                     SD = sd(Kappa))
+readr::write_tsv(summary.df, 
```

Comment:
I tend to prefer the aesthetic of splitting this type of thing out into two lines:

```R
summary.file <- file.path("results", "BRCA_train_3_models_summary_table.tsv")
readr::write_tsv(summary.df, summary.file)
```
I think it looks cleaner than the indent. I don't think the style guide says anything about this though.